Description: Disable ecore_econ tests that fail under buildd/sbuild
 These tests pass under pbuilder but fail with buildd/sbuild.  It may
 be related to home dir permissions - the other ecore_con tests remove
 the HOME env var.
Author: Ross Vandegrift <ross@kallisti.us>
Origin: other
Forwarded: no
Last-Update: 2014-04-23

--- a/src/tests/ecore_con/ecore_con_test_ecore_con.c
+++ b/src/tests/ecore_con/ecore_con_test_ecore_con.c
@@ -560,10 +560,10 @@
 void ecore_con_test_ecore_con(TCase *tc)
 {
    tcase_add_test(tc, ecore_test_ecore_con_init);
-   tcase_add_test(tc, ecore_test_ecore_con_local_user);
-   tcase_add_test(tc, ecore_test_ecore_con_local_user_home);
-   tcase_add_test(tc, ecore_test_ecore_con_local_user_tmp);
-   tcase_add_test(tc, ecore_test_ecore_con_local_user_none);
+   /* tcase_add_test(tc, ecore_test_ecore_con_local_user); */
+   /* tcase_add_test(tc, ecore_test_ecore_con_local_user_home); */
+   /* tcase_add_test(tc, ecore_test_ecore_con_local_user_tmp); */
+   /* tcase_add_test(tc, ecore_test_ecore_con_local_user_none); */
    tcase_add_test(tc, ecore_test_ecore_con_local_system);
    tcase_add_test(tc, ecore_test_ecore_con_local_system_fullpath);
    tcase_add_test(tc, ecore_test_ecore_con_local_system_negport);
